spec/dummy/db/migrate/20231218165617_add_searchable_content.rb

```diff
@@ -0,0 +1,16 @@
+class AddSearchableContent < ActiveRecord::Migration[6.0]
+  def up
+    unless column_exists? :pg_search_documents, :searchable_content
+      execute %|ALTER TABLE pg_search_documents
+        ADD COLUMN searchable_content tsvector GENERATED ALWAYS AS (
+          to_tsvector('#{Alchemy::PgSearch.config[:dictionary]}', coalesce(content, ''))
+        ) STORED;|
```

Comment:
Can we try to use Rails 7 feature of adding a generated column?

```suggestion
    add_column :pg_search_documents, :searchable_content, :virtual, type: :tsvector,
      as: "to_tsvector('#{Alchemy::PgSearch.config.fetch(:dictionary, "simple")}', coalesce(content, ''))",
      stored: true,
      if_not_exists: true
```